lib/bundler/source/git.rb

```diff
@@ -170,6 +170,9 @@ def install(spec, force = false)
           serialize_gemspecs_in(install_path)
           @copied = true
         end
+
+        return nil if gem_build_complete?(spec.extensions_dir)
```

Comment:
arg, this seems to have broken more things :(
@indirect do we need to run `git clean -xdf` before switching branches?